src/v3/src/components/Widget/index.tsx

```diff
@@ -255,6 +256,7 @@ export const Widget: FunctionComponent<WidgetProps> = (widgetProps) => {
     if ([IdxStatus.TERMINAL, IdxStatus.SUCCESS].includes(idxTransaction.status)
       || idxTransaction.nextStep?.name === IDX_STEP.SKIP // force safe mode to be terminal
       || !idxTransaction.nextStep) {
+      setCookieUserAuthenticated();
```

Comment:
Based on where you placed this function in Gen 2, I don't believe this is the correct place for Gen 3. In Gen 2 you added it to the global Success function, the equivalent location for this in Gen 3 is [here](https://github.com/okta/okta-signin-widget/blob/master/src/v3/src/OktaSignIn/index.ts#L158-L161). 

This section of the code not only triggers for successful authentications but also for terminal errors (when the user has not authenticated).

Reply:
fixed: https://github.com/okta/okta-signin-widget/pull/3434/commits/e0ff2c0d00c8b9d7301fde3c0bb75d9b516ccbd9